raiden/storage/sqlite.py

```diff
@@ -1460,6 +1461,19 @@ def update_light_client_protocol_message_set_signed_data(self, payment_id, msg_o
             last_id = cursor.lastrowid
         return last_id
 
+    def update_stored_msg_set_signed_tx_by_message_id(self, signed_tx: SignedTransaction, message_id: MessageID):
```

Comment:
Is this really needed? i mean we need to have the signed transaction logged on the database for some reason or we could just send it. I don't know what are the security implications of having signed messages on the database, i know that we do that in other cases, but here if having the signed message on the database is not really necessary maybe we can avoid store this on database.

Reply:
It's something I talked with @marcosmartinez7. It's a way of differentiating if a message has been processed or if it not.